maintaining-and-tuning/monitoring.md

```diff
@@ -162,15 +162,45 @@ threadpool_threads{job="grr_worker"}
 ```	
 rate(grr_client_crashes_total{job="grr_worker"}[5m])
 ```
-### Example visualization and alerting setup
+
+## Real-World Setup
+Although Prometheus can display basic charts using the
```

Comment:
I think we should describe Grafana not as an option but as something that a user wants to use to display Prometheus graphs. I.e. at the top of the page we can put something: "This document describes how to set up Prometheus and Grafana to monitor GRR server components."